inst/rmarkdown/templates/xaringan/resources/metropolis.css

```diff
@@ -32,26 +32,24 @@ border-bottom: 1px solid black;
 position: absolute;
 }
 
-.remark-slide-content > h2 {
+h2, h3, h4 {
```

Comment:
I'm not sure if this is a desired change: you removed the restriction of `>` (`h2` as a direct child of `.remark-slide-content`).

https://www.w3schools.com/cssref/css_selectors.asp

Reply:
Apparently it does not make a usage difference as `remark-slide-content` is anyway present in every slide.
However, idk if its "bad-style" or could potentially brake something so I added it again. 
Idk if that change was on purpose or not (I guess the latter).